JSDemos/Demos/DateBox/Formatting/description.md

```diff
@@ -1 +1,17 @@
-This demo shows how to&nbsp;use a&nbsp;locale-dependent format (or&nbsp;default format), <a href="/Documentation/ApiReference/Common/Object_Structures/format/#type">built-in format</a>, <a href="/Documentation/Guide/Common/Value_Formatting/#Format_Widget_Values/Custom_Format_String">LDML&nbsp;pattern</a>, and a&nbsp;format with literal characters to&nbsp;specify the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat">displayFormat</a>. Note that when the <a href="/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#useMaskBehavior">useMaskBehavior</a> option is&nbsp;set to&nbsp;<b>true</b>, the <b>DateBox</b> formats the display value and also controls user input.
\ No newline at end of file
+To customize the **DateBox** format, use the [displayFormat](/Documentation/ApiReference/UI_Widgets/dxDateBox/Configuration/#displayFormat) property. This demo illustrates the following use cases of this property:
+
+- **Locale-dependent format**     
+The DateBox formats its value according to the user's locale. This is the default behavior. Refer to the following help topic for more information: [Localize Dates, Numbers, and Currencies](/Documentation/Guide/Common/Localization/#Localize_Dates_Numbers_and_Currencies)
```

Comment:
I would link all these items to property values. So in this case, what should I use as the "displayFormat" property value. Is that an empty string, or something else? 

Let's say I want a switch in my application that changes between locale-dependent and some predefined format. When I revert to locale-dependent, what value should I use?

Reply:
As far as I understand the locale-dependent format (and according to the demo), the "displayFormat'' property is not defined as the DateBox automatically detects the user's locale and formats the date/time accordingly.  So in the above-mentioned scenario, we would need to either define the "displayFormat" property or remove it altogether.